src/ontology/cl-edit.owl

```diff
@@ -32194,6 +32196,30 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30382198") oboInOwl:hasE
 
+# Class: obo:CL_4042014 (spiny VIP neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35418660") obo:IAO_0000115 obo:CL_4042014 "A vip GABAergic cortical interneuron with a soma located in L1-3 of the neocortex in Mmus. This neuron has a multipolar morphology with spiny dendrites concentrating on L1 of the cortex and has a burst firing electrophysiological signature with highly dynamic dendritic spines.")
+AnnotationAssertion(terms:contributor obo:CL_4042014 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042014 "2024-07-11T12:09:53Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35418660") oboInOwl:hasRelatedSynonym obo:CL_4042014 "multipolar VIP neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35418660") oboInOwl:hasRelatedSynonym obo:CL_4042014 "superficial VIP-positive spiny neuron")
+AnnotationAssertion(rdfs:label obo:CL_4042014 "spiny VIP neuron"@en)
+SubClassOf(obo:CL_4042014 obo:CL_4023016)
+
+# Class: obo:CL_4042015 (VIP-ChAT interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30382198") Annotation(oboInOwl:hasDbXref "PMID:31754098") Annotation(oboInOwl:hasDbXref "PMID:32027647") Annotation(oboInOwl:hasDbXref "PMID:33301603") obo:IAO_0000115 obo:CL_4042015 "A vip GABAergic cortical interneuron expressing vasoactive intestinal polypeptide and choline acetyltransferase in the Mmus neocortex. This interneuron releases both γ-aminobutyric acid and acetylcholine.")
+AnnotationAssertion(terms:contributor obo:CL_4042015 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042015 "2024-07-11T12:22:48Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33301603") oboInOwl:hasExactSynonym obo:CL_4042015 "VIP+/ChAT+ interneuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32027647") oboInOwl:hasExactSynonym obo:CL_4042015 "VIP/ChAT interneuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32027647") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042015 "VChI")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33301603") oboInOwl:hasRelatedSynonym obo:CL_4042015 "VIP+/ChAT+ cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30382198") Annotation(oboInOwl:hasDbXref "PMID:32027647") Annotation(oboInOwl:hasDbXref "PMID:33301603") rdfs:comment obo:CL_4042015 "Work in the mouse barrel cortex and somatosensory cortex shows that a VIP-ChAT interneuron is predominantely located in L2/3 and more rarely in the deeper layers of the cortex (Dudai et al., 2021; Dudai et al., 2020). This neuronal type could be transcriptomically distinct as transcriptomic studies have identified VIP neurons expressing ChAT in a whole brain mouse transcriptomic study (Tasic et al., 2018).")
+AnnotationAssertion(rdfs:label obo:CL_4042015 "VIP-ChAT interneuron"@en)
+SubClassOf(obo:CL_4042015 obo:CL_4023016)
```

Comment:
Do you think this could be an Equivalent to?
'[vip GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023016)' and ('capable of' some '[acetylcholine secretion, neurotransmission](http://purl.obolibrary.org/obo/GO_0014055)')